### ISSUE TITLE
Game Client minimizes every time kolbot loads

### ISSUE THREAD

**Issue by XDarkwolf21** [kolton/d2bot-with-kolbot#1629]
My game Client fully minimizes each time kolbot loads whether it's mh or actual runs. Every game I join, everytime. Anyone be able to help me with this?

**Comment by DarkHorseDre** [comment]
check no.18 on the profile settings:
https://github.com/kolton/d2bot-with-kolbot/wiki/D2Bot-%23-Manager-Setup

**Comment by XDarkwolf21** [comment]
Perfect, thank you!

**Comment by DarkHorseDre** [comment]
your first response was "perfect thankyou" - what happened?! :)

what about the settings section of the same page, no4?

which parameters are you using? (profile no9)

**Comment by XDarkwolf21** [comment]
Yeah i thought for sure that would clear up the issue when I responded but it did not. I'm currently not home but I'll check my settings and report back.
